invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/ErrorMessage.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Message } from "semantic-ui-react";
+
+export class ErrorMessage extends Component {
+  render() {
+    const { error } = this.props;
+    return (
+      <Message className="ml-20 mr-20" negative>
+        <Message.Header>{error.message}</Message.Header>
+        {error.errors && (
+          <Message.List>
+            {error.errors.map((error) => {
```

Comment:
wouldn't it be easier to have a small serializer function and use just generic message component (accepting string) instead of making this specialised one ?

Reply:
We basically made the same component, independently!

can be made much simpler with some semantic-ui shorthand syntax, which i have done already in my PR.